client/src/components/Menu.jsx

```diff
@@ -55,7 +58,7 @@ class Menu extends React.Component {
 
   handleFilterBtnClick(event) {
     let filters = Object.assign({}, this.state.selectedFilters);
-    let targetFilter = event.target.parentElement.querySelector('.filterTitle');
+    let targetFilter = event.target.parentElement.querySelector(`.${filterTitle}`);
```

Comment:
is filterTitle a variable? Why does it need a ${}?

Reply:
Destructured another CSS style module to be able to toggle a class on a child component. That's what filterTitle is.